BUILD.gn

```diff
@@ -31,8 +33,18 @@ if (is_win) {
 
 config("vulkan_internal_config") {
   defines = [ "VULKAN_NON_CMAKE_BUILD" ]
+  cflags = [
+   "-Wno-conversion",
+   "-Wno-extra-semi",
+  ]
```

Comment:
Seems to also affect Android/Chromium.  Has this been compiled for each?

Reply:
I haven't - sounds like @mikes-lunarg can help test the GN build.